src/pages/messages/components/NviCandidatesNavigationAccordion.tsx

```diff
@@ -94,157 +60,77 @@ export const NviCandidatesNavigationAccordion = () => {
             </>
           ) : (
             <>
+              <Typography fontWeight="bold">{t('tasks.nvi.nvi_reporting_status')}</Typography>
+              <Box sx={{ display: 'flex', gap: '0.5rem' }}>
+                <LinearProgress
+                  aria-labelledby="progress-label"
+                  variant="determinate"
+                  value={nviCompletedPercentage}
+                  sx={{
+                    flexGrow: '1',
+                    my: '0.175rem',
+                    height: '1rem',
+                    bgcolor: 'white',
+                    borderRadius: '0.5rem',
+                    '& .MuiLinearProgress-bar': {
+                      backgroundColor: 'info.main',
+                    },
+                  }}
```

Comment:
LinearProgress har en prop for dette: https://mui.com/material-ui/api/linear-progress/#props
```suggestion
                  }}
                  color="info"
```